### ISSUE TITLE
Adjust BNB bunch timing to agree with MicroBooNE and ICARUS measurements

### ISSUE THREAD

**Issue by jzettle** [SBNSoftware/icaruscode#703]
Want to change the BNB bunch structure in our BNB generator to 18.936 ns following MicroBooNE's paper: https://arxiv.org/abs/2304.02076 and the ICARUS studies which agree with this. There is a corresponding SBND PR here that makes this change: https://github.com/SBNSoftware/sbndcode/pull/419

**Comment by PetrilloAtWork** [comment]
It would be good to have in `sbncode` a common source for this parameter (and maybe for most of BNB@GENIE configuration).

**Comment by jzettle** [comment]
I agree @PetrilloAtWork. The only reason I didn't say that was I wasn't sure how we wanted to handle that moving forward. We can think to use this as a test case for the idea of moving common SBND/ICARUS fcl parameters to `sbncode` if we think that is a good idea.

**Comment by jzettle** [comment]
I have added the idea to confirm the GENIE interaction time works as we expect. From looking at a test sample I made recently we thought further checking was needed
